Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -627,6 +627,8 @@ typedef NS_OPTIONS(NSUInteger, _WKWebViewDataType) {
 @property (nonatomic, copy, setter=_setEphemeralUIEventAttribution:) UIEventAttribution *_ephemeralUIEventAttribution WK_API_AVAILABLE(ios(16.0));
 - (void)_setEphemeralUIEventAttribution:(UIEventAttribution *)attribution forApplicationWithBundleID:(NSString *)bundleID WK_API_AVAILABLE(ios(16.0));
 
+@property (nonatomic, readonly, getter=_isBlockedByScreenTime) BOOL _isBlockedByScreenTime WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
```

Comment:
`getter=_isBlockedByScreenTime` is unnecessary since this is `readonly` and the name already matches.